### PR TITLE
add sam

### PR DESCRIPTION
**Pull request by danmoseley**
I actually didn't test this, maybe I should do that before it's merged..

### CODE REVIEW

File: pets.json [hunk 1]
```diff
@@ -18,5 +18,15 @@
     "github": "chsienki",
     "instagram": "brotherlucca",
     "twitter": ""
+  },
+  {
+    "name": "Sam",
+    "jobTitle": "Libraries Engineer",
+    "img": "https://github.com/danmoseley/petsof.net/blob/main/sam.jpg",
```
Comment (ericstj) [inline pets.json]: I think this should be `https://github.com/danmoseley/petsof.net/raw/main/sam.jpg`  The above link doesn't go to actual file.

Comment (chsienki) [reply pets.json]: Yeah, was just about to say the same thing.

Comment (joperezr) [reply pets.json]: Or you can add `raw=true` query param to the link you have which should work too. Try opening a file with this in a browser and you'll see the right file gets loaded:
```html
<html>
<img src="https://github.com/danmoseley/petsof.net/blob/main/sam.jpg?raw=true" />
</html>
```

Comment (carlossanlop) [reply pets.json]: Maybe not necessary. Look how the favicon is consumed:

https://github.com/chsienki/petsof.net/blob/cd78dabbb3806a2180925dc3ddf82ccfad7db6d4/template.html#L8

You can probably host it directly in the img folder.

Comment (chsienki) [reply pets.json]: It doens't quite work like that, it actually reads the URL and base64 encodes into the HTML file, which is then uploaded to azure where its served from.

Comment (danmoseley) [reply pets.json]: You expect me to test my changes? 😲

**Pull request merged**